src/service/index.ts

```diff
@@ -1,14 +1,20 @@
-import { api } from "./api"
+import { userChannel, repositoryChannel } from "./api"
 
 // https://api.github.com/users/tiagolopesdev
 // https://api.github.com/users/tiagolopesdev/repos
+// https://api.github.com/repos/tiagolopesdev/cobranca/commits
+
+export const getCommitsByRepository = async (nickName: string, nameRepository: string) => {
+    return await repositoryChannel.get(`${nickName}/${nameRepository}/commits`)
```

Comment:
If you are using the `async / await` sintaxe, you don't need to use the `then` method. Try to assign the return of the `repositoryChannel.get()` method into a variable and then use the value as you need

Reply:
What the best cenary to use then. You can send a link to read about?